tests/driver_mcp23x17/Makefile

```
@@ -0,0 +1,10 @@
+include ../Makefile.tests_common
```

```suggestion
include ../Makefile.drivers_common
```

The test have been restructured in the meantime and now this is the common Makefile for all tests of drivers.